src/api/app/controllers/webui/appeals_controller.rb

```diff
@@ -18,7 +18,7 @@ def create
     @decision = Decision.find(decision_params)
     @appeal = Appeal.new(appeal_params)
     @appeal.decision = @decision
-    @appeal.appellant = User.session!
```

Comment:
If there is no logged-in user, then the authorization check will fail. This is what we want.